packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -289,12 +289,12 @@ const TrackListItemComponent = (props: TrackListItemComponentProps) => {
   const handleOpenOverflowMenu = useCallback(() => {
     const overflowActions = [
       OverflowAction.SHARE,
-      !isTrackOwner && !isLocked && (!isUnlisted || has_current_user_saved)
+      !isTrackOwner && !isLocked && !isUnlisted
```

Comment:
isn't this undoing the changes you did last week? i thought we wanted to let the user unfavorite if they had already favorited.

Reply:
product team decided against this in last sync